release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -291,6 +291,11 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+[id="tuning-of-power-states"]
+==== Power saving configurations
+
+In {product-title} {product-version}, by enabling C-states and OS-controlled P-states, you can use different power saving configurations for critical and non-critical workloads through the new `perPodPowerManagement` workload hint, the `cpu-c-states.crio.io` and `cpu-freq-governor.crio.io` CRI-O annotations. For more information about the feature, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#node-tuning-operator-pod-power-saving-config_cnf-master[Power saving configurations].
```

Comment:
@mrniranjan Rephrased the second sentence a bit to avoid passive voice.
> In {product-title} {product-version}, by enabling C-states and OS-controlled P-states, you can use different power saving configurations for critical and non-critical workloads. 
> You can apply the configurations through the new `perPodPowerManagement` workload hint, the `cpu-c-states.crio.io` and `cpu-freq-governor.crio.io` CRI-O annotations.